book/chapter-00.md

```diff
@@ -26,30 +26,30 @@ We can use these tools to push forth a variety of good and bad outcomes:
 | Open access           | Censorship          |
 | Fast finality         | Slow finality       |
 | Decentralized control | Centralized control |
-| Inexpensive           | Expensive           |
+| Low cost              | High cost           |
 
-While we can use cryptoeconomics to analyze centralized systems, these systems were not built with cryptoeconomics in mind. The constraints on power in a centralized payment processor, for instance, mostly derive from the legal system and reputational costs. There is nothing physically preventing a central payment processor from stealing or losing user's funds. This changes when we introduce cryptoeconomic designs which limit the power of central authorities.
+While we can use cryptoeconomics to analyze centralized systems, these systems were not built with cryptoeconomics in mind. The constraints on power in a centralized payment processor, for instance, mostly derive from the applicable legal system and reputational costs. There is nothing physically preventing a central payment processor from stealing or losing a user's funds. This changes when we introduce cryptoeconomic designs, which limit the power of central authorities.
 
-That’s cool… but how do we govern and design decentralized networks? With these open networks, we want to be able to open participation to the public internet, and incentivize participation in the security and maintenance of the network (running nodes, mining, etc).
+That’s cool… but how do we govern and design decentralized networks? With these open networks, we want to be able to open participation to the public internet and incentivize participation in the security and maintenance of the network (running nodes, mining, etc).
 
-In a centralized payment processor, we trust the central authority  to verify transactions and update balances. However, in a decentralized payment processor, everyone runs everything by listening for and downloading new transactions. But in what order? Lack of agreement on ordering leads to a potential problem which prevents everyone from agreeing on a common state of the network.
+In a centralized payment processor, we trust the central authority to verify transactions and update balances. However, in a decentralized payment processor, everyone runs everything by listening for and downloading new transactions. But in what order? Lack of agreement on the order of transactions leads to a potential problem - the inability of everyone to agree on a common state of the network.
 
   
```

Comment:
Maybe it's just me, but I had to pause longer than would be ideal to work out what's going on in the table.  Consider adding a third row to the table and making it explicit that Aparna is trying to execute two txs between the initial state and the final state. 

So if you're okay with that suggestion, the revised table would look something like this [actually, I only revised the left side of the table (under Order 1); I haven't figured out how the formatting works enough to edit the right side]--
   
 |               | Order 1                                                             | Order 2                                                              |
 |---------------|---------------------------------------------------------------------|----------------------------------------------------------------------|
 | Initial State | Aparna has 10 eth |
 | Transactions   | Aparna first sends Jing 8 eth, and then Aparna sends Karl 8 eth | Aparna has 2 eth, Karl has 8 eth, Jing has 0 eth         |
 | Final State   | Aparna has 2 eth, Jing has 8 eth, Karl has 0 eth                    | Aparna has 2 eth, Karl has 8 eth, Jing has 0 eth         |